### ISSUE TITLE
Add MTB Map to Taginfo Projects

### ISSUE THREAD

**Issue by danieldegroot2** [utagawal/MTBmaptiles#19]
Show the OpenStreetMap community what [data](https://wiki.openstreetmap.org/wiki/Tags) you are using. Add MTB Map to the [list](https://taginfo.openstreetmap.org/projects) of [Taginfo Projects](https://github.com/taginfo/taginfo-projects).

**Comment by utagawal** [comment]
Good idea. Any possibility to generate this automlatically for fetching the data? Feel free to submit a pull request.

**Comment by danieldegroot2** [comment]
You can possibly write a script to get (update) the data. I can't help you with this, however.

**Comment by danieldegroot2** [comment]
Opened also an issue for Opentraveller https://github.com/Traveller31/opentraveller-carto/issues/1
